backend/mora/service/address.py

```diff
@@ -312,6 +312,15 @@ def get_one_address(c, addrrel, class_cache=None):
     scope = util.checked_get(addrclass, 'scope', 'DAR')
 
     if scope == 'DAR':
+        def make_error_object(errordesc):
```

Comment:
Why not replace _all_ of the places we create these address objects? Specifically lines 355 and 392. They all need to have the exact same fields (except for the addition of `ERROR` which is optional), so why leave it up to chance by constructing the dictionaries by each time?